scripts/variantstore/wdl/GvsQuickstartIntegration.wdl

```diff
@@ -74,22 +66,62 @@ workflow GvsQuickstartIntegration {
         call QuickstartVcfIntegration.GvsQuickstartVcfIntegration as QuickstartVcfVQSRLiteIntegration {
             input:
                 branch_name = branch_name,
+                is_wgs = true,
                 use_VQSR_lite = true,
                 extract_do_not_filter_override = false,
                 dataset_suffix = "lite_vcf",
-                gatk_override = BuildGATKJar.jar,
+                use_default_dockers = use_default_dockers,
+                gatk_override = if (use_default_dockers) then none else BuildGATKJar.jar,
                 interval_list = FilterIntervalListChromosomes.out,
                 expected_output_prefix = expected_output_prefix,
         }
         call QuickstartVcfIntegration.GvsQuickstartVcfIntegration as QuickstartVcfVQSRClassicIntegration {
             input:
                 branch_name = branch_name,
+                is_wgs = true,
                 use_VQSR_lite = false,
                 extract_do_not_filter_override = true,
                 dataset_suffix = "classic_vcf",
-                gatk_override = BuildGATKJar.jar,
+                use_default_dockers = use_default_dockers,
+                gatk_override = if (use_default_dockers) then none else BuildGATKJar.jar,
                 interval_list = FilterIntervalListChromosomes.out,
                 expected_output_prefix = expected_output_prefix,
         }
     }
+
+    if (run_exome_integration) {
+        call QuickstartVcfIntegration.GvsQuickstartVcfIntegration as QuickstartVcfExomeIntegration {
+            input:
+                branch_name = branch_name,
+                is_wgs = false,
+                use_VQSR_lite = true,
+                extract_do_not_filter_override = false,
+                dataset_suffix = "exome",
+                use_default_dockers = use_default_dockers,
+                gatk_override = if (use_default_dockers) then none else BuildGATKJar.jar,
+                interval_list = full_exome_interval_list,
```

Comment:
Note that we aren't subsetting the Exome interval list down to 20, X, and Y. Not sure if I should do so. Not sure why I didn't. Discuss.

Reply:
how many intervals are even on chrX and chrY in an exome haha

but like how much time does this even save us?